integration-tests/bats/import-update-tables.bats

```diff
@@ -14,6 +14,14 @@ CREATE TABLE test (
   c5 BIGINT COMMENT 'tag:5',
   PRIMARY KEY (pk)
 );
+SQL
+
+    cat <<SQL > 1pk1col-char-sch.sql
+CREATE TABLE test (
+  pk BIGINT NOT NULL COMMENT 'tag:0',
+  c char(5) COMMENT 'tag:1',
```

Comment:
Just a small nitpick: let's uppercase `char` here since we're uppercasing all of the other type declarations.

Reply:
Ah true - thanks.